spec/index.bs

```diff
@@ -128,20 +128,24 @@ The <dfn>contact picker task source</dfn> is a [=task source=].
 A <dfn>user contact</dfn> consists of:
 <div dfn-for="user contact">
 
-* <dfn>names</dfn>, a [=list=] of {{DOMString}}s, initially empty, each [=list/item=] representing
-  a unique name corresponding to the user.
-* <dfn>emails</dfn>, a [=list=] of {{DOMString}}s, initially empty, each [=list/item=] representing
-  a unique [=valid e-mail address=] of the user.
-* <dfn>numbers</dfn>, a [=list=] of {{DOMString}}s, initially empty, each [=list/item=]
-  representing a unique phone number of the user.
-* <dfn>addresses</dfn>, a [=list=] of {{ContactAddress}}es, initially empty, each [=list/item=]
-  representing a unique [=physical address=] of the user.
+* <dfn>names</dfn>, a [=list=] of {{DOMString}}s, each [=list/item=] representing a unique name
+  corresponding to the user.
+* <dfn>emails</dfn>, a [=list=] of {{DOMString}}s, each [=list/item=] representing a unique
+  [=valid e-mail address=] of the user.
+* <dfn>numbers</dfn>, a [=list=] of {{DOMString}}s, each [=list/item=] representing a unique phone
+  number of the user.
+* <dfn>addresses</dfn>, a [=list=] of {{ContactAddress}}es, each [=list/item=] representing a
+  unique [=physical address=] of the user.
+* <dfn>icons</dfn>, a [=list=] of {{Blob}}s, each [=list/item=] representing a unique image of the
+  user.
```

Comment:
What could these be used for? Are there implementation-specific considerations? Will they be square? Circular? Is it fine for the browser to just return what the OS says?

I'd love to see, for each property, _(a)_ example use-cases, and _(b)_ implementation and usage considerations, capturing questions like these.

Reply:
You'll find those concerns addressed here: https://github.com/WICG/contact-api/issues/13#issuecomment-545895103.

I didn't add an example for icons since I already added one for addresses, and they are pretty much the same. I can add yet another example if you feel that would help.

Reply:
The use-cases and reasons can still be listed, right?